src/documentation/domains/issue/controller/issue.controller.docs.ts

```diff
@@ -1,9 +1,35 @@
 /**
  * @swagger
  * paths:
+ *   /api/issue/{issueId}/resume:
```

Comment:
Please add security to the endpoint, it cannot be tested in swagger.

Reply:
Done.